app/components/question-box.jsx

```diff
@@ -12,6 +13,10 @@ const QuestionBox = props => {
     description = '',
     participants = 0,
     contentAlign = 'center',
+<<<<<<< HEAD
```

Comment:
These lines with the + are from the git merge, these are lines of code that it couldn't resolved. Please fix up the code choosing what should remain in the file. And make sure it compiles and runs.